src/containers/Test/index.tsx

```diff
@@ -0,0 +1,25 @@
+'use client';
+
+import React from 'react';
+import { requestNotificationPermission } from '@/services/firebase';
+import CButton from '@/components/CButton';
+
+const Test = () => {
+  const handleClick = () => {
+    requestNotificationPermission();
```

Comment:
Call this in a file that if the page loads, it calls this function. 

I don't know what is the appropriate file is in Next 14 though